cyvlfeat/fisher/cyfisher.pyx

```diff
@@ -20,34 +20,57 @@ from cyvlfeat._vl.fisher cimport VL_FISHER_FLAG_FAST
 
 
 @cython.boundscheck(False)
-cpdef cy_fisher(float[:, :] X,
-                float[:, :] MEANS,
-                float[:, :] COVARIANCES,
-                float[:] PRIORS,
+cpdef cy_fisher(np.ndarray X,
```

Comment:
Do you suggests that there should be two versions of the function ? cy_fisher_f and cy_fisher_d ?